Library/Homebrew/formula_creator.rb

```diff
@@ -9,11 +9,14 @@ module Homebrew
   #
   # @api private
   class FormulaCreator
-    attr_reader :args, :url, :sha256, :desc, :homepage
+    attr_reader :url, :sha256, :desc, :homepage
     attr_accessor :name, :version, :tap, :mode, :license
```

Comment:
Might be nice to make as many of these accessors as possible and assigned in the constructor. From a quick scan: looks like `name` is probably the only one that needs to be assigned from outside the class?

Reply:
Good catch. Will fix.